lms/static/certificates/sass/_build.scss

```diff
@@ -1,45 +1,53 @@
 // ------------------------------
```

Comment:
NOTE: revamping this shared build compile list to use static version of PL assets.